plugin/src/gatsby-node.ts

```diff
@@ -200,19 +201,22 @@ export function createResolvers(
   { downloadImages, typePrefix = "" }: ShopifyPluginOptions
 ) {
   if (!downloadImages) {
-    const resolvers = {
-      [`${typePrefix}ShopifyProductImage`]: {
-        gatsbyImageData: getGatsbyImageResolver(resolveGatsbyImageData),
-      },
-
-      [`${typePrefix}ShopifyProductFeaturedImage`]: {
-        gatsbyImageData: getGatsbyImageResolver(resolveGatsbyImageData),
-      },
-
-      [`${typePrefix}ShopifyCollectionImage`]: {
-        gatsbyImageData: getGatsbyImageResolver(resolveGatsbyImageData),
-      },
-    };
+    const imageNodeTypes = [
+      `ShopifyProductImage`,
+      `ShopifyProductFeaturedImage`,
+      `ShopifyCollectionImage`,
```

Comment:
Not part of this PR, but should we add a resolver for `ShopifyCollectionImage` if they are not including collections? It might not actually do anything but I noticed we are not checking if collections are included in their ShopifyConnections plugin config.

Reply:
You're right, it shouldn't be there in that case